packages/embark-ui/src/actions/index.js

```diff
@@ -82,6 +82,17 @@ export const blocks = {
   failure: (error) => action(BLOCKS[FAILURE], {error})
 };
 
+export const BLOCKS_FULL = createRequestTypes('BLOCKS_FULL');
+export const blocksFull = {
+  request: (from, limit) => {
+    const txObjects = true;
+    const txReceipts = true;
+    return action(BLOCKS_FULL[REQUEST], {from, limit, txObjects, txReceipts});
```

Comment:
Fluff, but I'd prefer:
```suggestion
    return action(BLOCKS_FULL[REQUEST], {from, limit, txObjects: true, txReceipts: true});
```
as it gets rid of two useless variables

Reply:
Done.